scout/server/blueprints/variants/controllers.py

```diff
@@ -439,6 +439,71 @@ def parse_variant(
     return variant_obj
 
 
+def download_str_variants(case_obj, variant_objs):
+    """Download filtered STR variants for a case to an excel file
+
+    Args:
+        case_obj(dict)
+        variant_objs(PyMongo cursor)
+
+    Returns:
+        an HTTP response containing a csv file
+    """
+
+    def generate(header, lines):
+        yield header + "\n"
+        for line in lines:
+            yield line + "\n"
+
+    DOCUMENT_HEADER = [
+        "Index",
+        "Repeat locus",
+        "Reference repeat unit",
```

Comment:
```suggestion
        "Repeat unit",
```
since you very nicely pick the display ru if available!